MANIFEST.in

```diff
@@ -2,6 +2,6 @@ include pyopenjtalk/version.py
 exclude .gitignore .gitmodules .travis.yml release.sh tox.ini
 prune .github/*
 prune docs/**
-recursive-include lib *.cpp *.c *.h README LICENSE COPYING CMakeLists.txt *.in
+recursive-include lib *.cpp *.c *.h README COPYING CMakeLists.txt *.in
```

Comment:
Get the warning: `warning: no files found matching 'LICENSE' under directory 'lib'`.
This is because there is no file called `LICENSE` in the `lib` directory.